src/T2DMap.cpp

```diff
@@ -4583,11 +4585,11 @@ void T2DMap::slot_setCustomLine()
     connect(dialog, SIGNAL(rejected()), this, SLOT(slot_cancelCustomLineDialog()));
 
     QStringList lineStyles;
-    lineStyles << "solid line"
-                << "dot line"
-                << "dash line"
-                << "dash dot line"
-                << "dash dot dot line";
+    lineStyles << QStringLiteral("solid line")
```

Comment:
Please undo/revert these lines as I will be blowing them away in the next 24 Hours...